dangerfile.js

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { danger, warn, fail } from 'danger';
+
+const hasCHANGELOGChanges = danger.git.modified_files.some((filePath) => {
+  const srcFilePattern = /CHANGELOG.md/i;
+  return srcFilePattern.test(filePath);
+});
+
+const hasModifiedFiles = danger.git.modified_files.some((filePath) => {
+  const filePattern = /(src)/;
```

Comment:
This should be scripts instead of src

Reply:
Updated: https://github.com/cerner/terra-aggregate-translations/pull/1/commits/1730c5af56deebfe97427c6e78bea6c0ea32fb47